### PR TITLE
Added LendMark

### PR DESCRIPTION
**Pull request by SlenkyDev**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/lendmark/manifest.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "%.lendmark.sk",
+      "%.lmstreamer.live"
+    ],
+    "social": {
+      "web": "https://lendmark.sk/",
+      "web_shop": "https://store.lendmark.sk/",
+      "discord": "https://discord.lendmark.sk",
+      "tiktok": "lendmark.sk",
+      "instagram": "lendmark.sk",
+      "youtube": "https://www.youtube.com/channel/UC1Khdip7ObGGN4OCsuSlg0gt"
```
Comment (JakobBruening) [inline minecraft_servers/lendmark/manifest.json:15]: The link does not work.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/lendmark/manifest.json (hunk 1 above)
Comment (SlenkyDev) [reply minecraft_servers/lendmark/manifest.json:15]: Which link exactly?

Comment (SlenkyDev) [reply minecraft_servers/lendmark/manifest.json:15]: got it

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Comment by SlenkyDev** [comment]
It looks like the images weren't updated

**Comment by JakobBruening** [comment]
Hey @SlenkyDev!
The logo is shown correctly on the LABY.net server page. But for the background we request `background@2x.png`, that was not added with your Pull request.

BR, Jakob

**Comment by SlenkyDev** [comment]
> Hey @SlenkyDev!
> The logo is shown correctly on the LABY.net server page. But for the background we request `background@2x.png`, that was not added with your Pull request.
> 
> BR, Jakob

Oh okay, can I open a second pull request to add it?

**Comment by JakobBruening** [comment]
Of course you can do that! I'm looking forward seeing the PR!

**Comment by SlenkyDev** [comment]
Okay, thanks!

**Comment by SlenkyDev** [comment]
> Of course you can do that! I'm looking forward seeing the PR!

https://github.com/LabyMod/server-media/pull/107

**Pull request merged**